notebooks/official/model_evaluation/custom_tabular_classification_model_evaluation.ipynb

```diff
@@ -0,0 +1,1639 @@
+{
```

Comment:
<p>Change to:</p><pre class="ql-syntax" spellcheck="false">!gsutil cp "model.joblib" {BUCKET_URI}/{MODEL_ARTIFACT_DIR}/
</pre><p>Added double quotations around model.joblib ow it'll give error that the url was not found!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1219/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='55'/>

Reply:
<p>added.</p>